arches/urls.py

```diff
@@ -312,6 +312,7 @@
     re_path(r"^etl-manager$", ETLManagerView.as_view(), name="etl_manager"),
     re_path(r"^clear-user-permission-cache", ClearUserPermissionCache.as_view(), name="clear_user_permission_cache"),
     re_path(r"^transform-edtf-for-tile", api.TransformEdtfForTile.as_view(), name="transform_edtf_for_tile"),
+    re_path(r"^api/spatialview/(?P<identifier>[a-f0-9\-]+|[\w-]+)/?$", api.SpatialView.as_view(), name="spatialview_api"),
```

Comment:
this does not allow a user to list all spatialviews they have permissions to see. Should also have the following

`re_path(r"^api/spatialview$", api.SpatialView.as_view(), name="spatialview_api"),`